.github/workflows/spelling.yml

```diff
@@ -0,0 +1,22 @@
+---
+name: Check for spelling with vale
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  vale:
+    name: Spelling
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Vale
+        uses: ConsenSys/docs-gha/spelling@main
```

Comment:
@m4sterbunny If you merge [#6 ](https://github.com/Consensys/docs-gha/pull/6) , or copy pasta the content as a branch directly into `docs-gha` ^ change the above line `main` to your branch's name and you can test things on this repo?